kong/api/routes/clustering.lua

```diff
@@ -0,0 +1,18 @@
+local clustering = require("kong.clustering")
+local kong = kong
+
+
+return {
+  ["/clustering/status"] = {
+    GET = function(self, db, helpers)
+      if kong.configuration.role ~= "admin" then
```

Comment:
Another question: Is there an endpoint for such status on the data-plane?
Things I'd look during debugging is last sync, last ping, which CP node is the data-plane connected to right now.

Reply:
No, however the DP does have pretty detailed logging explaining how things are going. But if there are enough use case we may add another API later on the DP side.

Reply:
We should have a programmatic way of knowing if a data-plane is healthy or not.
The `/status` endpoint on DP nodes will return `200` no matter what.

Reply:
That is generally hard to tell on the DP side, since DP does not necessarily know the state of the config, only CP does.

That being said, if there are enough interest we can certainly add this endpoint to DP in the future.